app/slack-responses/edit-initiative-element.ts

```diff
@@ -0,0 +1,74 @@
+import { TextElement, DialogError, SelectElement, SelectElementOption } from 'slack';
+import { InitiativeResponse, Status, getStatusDisplay } from '../initiative';
+
+export enum EditInitiativeFieldName {
+  INITIATIVE_NAME = 'initiative_name',
+  INITIATIVE_DESCRIPTION = 'initiative_description',
+  INITIATIVE_STATUS = 'initiative_status'
+}
+
+export enum EditInitiativeFieldError {
+  EMPTY_ERROR = 'This field cannot be empty',
+  UNCHANGED_ERROR = 'Please update a field'
+}
+
+export class SelectIniatitiveStatus implements SelectElementOption {
+  label: string;
+  value: string;
+  constructor(label: string, value: string) {
+    this.label = label;
+    this.value = value;
+  }
+}
+
+export class EditInitiativeStatus implements SelectElement {
+  label: string;
+  name: string;
+  value: string;
+  type: 'select';
+  options: SelectElementOption[];
+  constructor(initiative: InitiativeResponse) {
+    this.name = EditInitiativeFieldName.INITIATIVE_STATUS;
+    this.label = 'Select a stuatus for this initiative';
+    this.value = initiative.status;
+    this.options = Object.values(Status).map(status => new SelectIniatitiveStatus(getStatusDisplay(status), status));
+    this.type = 'select';
```

Comment:
is there an option for setting the default select option to be current status?
i know there is for general message blocks, not sure about modals

Reply:
that should be what line 33 is doing, setting the initial value in the dropdown to be the current status, similar to how the initial value in the name and description field with be the current values